packages/edit-site/src/components/sidebar/index.js

```diff
@@ -1,20 +1,35 @@
 /**
  * WordPress dependencies
  */
-import { createSlotFill, Panel } from '@wordpress/components';
+import { createSlotFill } from '@wordpress/components';
+import { ComplementaryArea } from '@wordpress/interface';
 import { __ } from '@wordpress/i18n';
+import { cog, pencil } from '@wordpress/icons';
 
 const { Slot: InspectorSlot, Fill: InspectorFill } = createSlotFill(
 	'EditSiteSidebarInspector'
 );
-
 function Sidebar() {
 	return (
-		<div className="edit-site-sidebar">
```

Comment:
@jorgefilipecosta want to check why we removed this, because I'm adding it back at https://github.com/WordPress/gutenberg/pull/20530/files?file-filters%5B%5D=.js#diff-3924a41ad88da62082971d8c0aa8b20fL14

The style.css is still targeting that class. I've only noticed after the changes made to the hooks (color, line-height), because some styles were leaking to the edit-site sidebar.